### PR TITLE
feat: send uiab experiment assignments in x-uiab header to graphql api

### PR DESCRIPTION
**Pull request by emuvente**

### CODE REVIEW

**Review by mcstover — APPROVED** [review]

File: src/api/ExperimentIdLink.js [hunk 1]
```diff
@@ -25,17 +25,24 @@ function buildExpHeaders(cache) {
...
+export default ({ cookieStore }) => {
 	return setContext((operation, previousContext) => {
-		// Fetch experiment header value
+		// Initialize new context
+		const newContext = { ...previousContext };
+		newContext.headers = newContext.headers ?? {};
+
+		// Fetch experiment header values
 		const experimentHeader = buildExpHeaders(previousContext?.cache);
+		const uiabValue = cookieStore.get(UIAB_COOKIE_NAME) ?? '';
```
Comment (dyersituations) [inline src/api/ExperimentIdLink.js:36]: Keep in mind that we don't actually store the assignments in the `uiab` cookie unless the assignment is forced via URL (or it's a user with an old `uiab` cookie from before the experimentation changes a few months ago).

Comment (emuvente) [reply src/api/ExperimentIdLink.js:36]: That's exactly what we need this for, passing the forced assignments to the api.

**Comment by emuvente** [comment]
Pairs with kiva/kiva#11802

File: src/api/ExperimentIdLink.js (hunk 1 above)
Comment (dyersituations) [reply src/api/ExperimentIdLink.js:36]: Cool, good to know!

**Review by dyersituations — APPROVED** [review]

**Pull request merged**